config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -248,6 +248,10 @@ endif
 
 if CHIP_WIFI
 
+config NRF700X_REV_A # we need this config until the revB RPU is supported
```

Comment:
It's ok. Another way could be adding select NRF700X_REV_A here: https://github.com/nrfconnect/sdk-connectedhomeip/blob/master/config/nrfconnect/chip-module/Kconfig.features#L22 where all other NRF7002 configs are set. I leave the decision to you :)